src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -2416,10 +2437,10 @@ TargetShardIntervalForFastPathQuery(Query *query, Const **partitionValueConst,
 		*isMultiShardQuery = true;
```

Comment:
small note: this line does not seem to be covered by tests. Can it ever happen?

Reply:
That's actually a good point, I think it is copy&paste from RouterPlanner, but at this point we're sure that it is `FastPathQuery`, so we cannot/should not have any multi-shard queries.

I can think a bit more and fix it on master? As I'm not very confident to remove it at this point